src/odemis/util/test/linalg_test.py

```diff
@@ -127,6 +128,36 @@ def test_are_collinear(self):
         non_linear_points = numpy.array([[1, 2, 12], [-1, -2, -4], [3, 4, 22]])
         self.assertFalse(are_collinear(non_linear_points[0], non_linear_points[1], non_linear_points[2]))
 
+    def test_generate_triangulation_points(self):
+        """
+        Test the number of focus points when given area changes.
+        """
+        max_dis = 450e-06
+        length_1_start = 0
+        length_1_end = 1.77e-03
+        length_2_start = 0
+        length_2_end = 1.77e-03
+
+        # given area is 100% of maximum area (1.77mm x 1.77mm)
+        given_area_coords = (length_1_start, length_2_start, length_1_end, length_2_end)
+        focus_points = generate_triangulation_points(max_dis, given_area_coords)
+        self.assertEqual(len(focus_points), 16)
+        self.assertEqual(len(focus_points[0]), 2)  # (x,y)
```

Comment:
You could try more explicit and easy checks for the things that are claimed in the docstring of the function. In particular:
* every focus point is within the the given area.
* for any point in the area, there is always a focus point which is closer than the max distance. (you can check the smallest distance to point p with `min(math.hypot(f[0]-p[0], f[1]-p[1] for f in focus_points))`). You could just hard-code a couple of "typical points" to test.